rosdep/base.yaml

```
@@ -1338,6 +1338,12 @@ glc:
   arch: [glc]
   gentoo: [media-libs/quesoglc]
   nixos: [quesoglc]
+glpk:
+  arch: [glpk-git]
+  debian: [libglpk-dev]
+  fedora: [glpk-devel]
```

```suggestion
  fedora: [glpk-devel]
  gentoo: [sci-mathematics/glpk]
```

From @mabelzhang 's research